devtools/cli/src/test/java/io/quarkus/cli/CliProjectJBangTest.java

```diff
@@ -76,7 +76,7 @@ public void testCreateAppOverrides() throws Exception {
                 "--package-name=custom.pkg",
                 "--output-directory=" + nested,
                 "--app-config=" + String.join(",", configs),
-                "-x vertx-web",
```

Comment:
Where does this `vertx-web` actually map?

Reply:
As a dependency in the source file

Reply:
Maybe clearer: this is a parameter passed to the CLI -- translated into a dependency in the jbang source file (because it doesn't have a pom.xml). The extension you're trying to add still has to exist. ;)

Reply:
Ok, I get it... `vertx-web` is translated to `quarkus-vertx-web`...